xblockutils/studio_editable.py

```diff
@@ -269,7 +269,8 @@ def validate(self):
         return validation
 
 
-class StudioContainerXBlockMixin:
+@XBlock.needs('mako')
+class StudioContainerXBlockMixin(XBlockMixin):
```

Comment:
@bradenmacdonald, it's very similar to the [StudioEditableBlock](https://github.com/openedx/edx-platform/blob/1950949c9edcd23c3172b11de2cdb607a996e00a/xmodule/studio_editable.py#L9). Would it make sense to  merge them?

Reply:
Yes. I think the `author_edit_view`/`author_preview_view` functionality is unique, but we could probably make this one so that it subclasses `StudioEditableBlock` and then we could remove the `render_children` from this one. Is that what you're thinking?

Reply:
@bradenmacdonald, I think that the `StudioEditableBlock` needs to be a subclass of the `StudioContainerXBlockMixin`. We cannot add an `edx-platform` import to the `xblock-utils`, because this library is used by the workbench.

Reply:
I see. So then this PR has to come first. I got the tests passing now so @Agrendalath  can you please give it another look?